src/main/scala/org/broadinstitute/hail/Utils.scala

```diff
@@ -967,23 +968,19 @@ object Utils extends Logging {
     val outputStream = destFS.create(destPath)
 
     try {
-      var i = 0
-      while (i < srcFileStatuses.length) {
-        val fileStatus = srcFileStatuses(i)
-        val lenAdjust: Long = if (isBGzip && i < srcFileStatuses.length - 1)
-          -28
-        else
-          0
```

Comment:
Tom, I just want to double-check the BGZFCodec doesn't write empty gzip blocks at the end of files by default, for example, when compressing the output of saveAsTextFile.  While it is BGZ compliant to have empty blocks, we found several tools that incorrectly treated them as premature end of files.